.infrastructure/st-configMap.yml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: mysql-config
+  namespace: mysql
+data:
+  init.sql: | 
+    GRANT ALL PRIVILEGES ON app_db.* TO 'app_user'@'%';
+    USE app_db;
+    CREATE TABLE lists {
```

Comment:
The CREATE TABLE statement uses curly braces, which is incorrect SQL syntax. Replace the curly braces with parentheses: CREATE TABLE lists ( ... ).